fhd_core/setup_metadata/fhd_struct_init_meta.pro

```diff
@@ -82,7 +95,7 @@ ENDIF ELSE BEGIN
     hist_A1=histogram(tile_A1,min=1,max=n_tile,/binsize,reverse_ind=ria)
     hist_B1=histogram(tile_B1,min=1,max=n_tile,/binsize,reverse_ind=rib)
     hist_AB=hist_A1+hist_B1
-    tile_names=layout.antenna_numbers
+    tile_names=indgen(n_tile)+1
```

Comment:
This seems wrong to me, I thought that tile_names in the FHD metadata reflected the true tile names (e.g. starting at 11) not the index...

Reply:
Good catch, that was an accidental push